OneCommunityGlobal/HGNRest
# Luis warning component backend

# Description
This is the backend for PR 1873
The backend created the warnings route, controller, and assigned a warnings array to the user's profile 
The frontend targets this endpoint to receive the user's current warnings. 

## Related PRS (if any):
To test this backend PR you need to checkout the #1873 frontend PR.
…

## Main changes explained:
-Created warningsController, warningRouter, and added an array of warnings to the user's profile
…

## How to test:
1. check into current branch
2. do `npm run build && npm run start` 
3. Follow instructions on PR 1873
4. You must check into this Backend to test the front end.


## Note:
